app/uk/gov/hmrc/identityverificationquestions/monitoring/metric/MetricsService.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.identityverificationquestions.monitoring.metric
+
+import com.codahale.metrics.Timer
+import com.kenshoo.play.metrics.Metrics
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class MetricsService @Inject()(metrics: Metrics) {
+
+  lazy val payeConnectorTimer: Timer = metrics.defaultRegistry.timer("pay-as-you-earn-emp-connector-response-timer")
```

Comment:
We may need to discuss this - not sure I understand why we are using timers here?  There are Meters, Counters, Derivatives, Ratios, etc available.  Why did you choose timer?  How will that work in Grafana to show us the 3 states?